packages/syft/src/syft/core/node/common/node_service/node_setup/node_setup_service.py

```diff
@@ -103,27 +104,34 @@ def get_setup(
     msg: GetSetUpMessage, node: DomainInterface, verify_key: VerifyKey
 ) -> GetSetUpResponse:
 
-    _current_user_id = msg.content.get("current_user", None)
+    _setup = model_to_json(node.setup.first(domain_name=node.name))
 
-    users = node.users
+    return GetSetUpResponse(
+        address=msg.reply_to,
+        content=_setup,
+    )
 
-    if not _current_user_id:
-        try:
-            _current_user_id = users.first(
-                verify_key=verify_key.encode(encoder=HexEncoder).decode("utf-8")
-            ).id
-        except Exception as e:
-            traceback_and_raise(e)
 
-    if users.role(verify_key=verify_key).name != "Owner":
-        raise AuthorizationError("You're not allowed to get setup configs!")
+def update_settings(
+    msg: UpdateSetupMessage, node: DomainInterface, verify_key: VerifyKey
+) -> UpdateSetupResponse:
+    if node.users.role(verify_key=verify_key).id == node.roles.owner_role.id:
+        if msg.domain_name:
+            node.name = msg.domain_name
```

Comment:
Should the `domain_name` be stored in the db?

Reply:
Yup!